marketplace_builder/views/pages/tutorials/users/users.liquid

```diff
@@ -9,26 +9,43 @@ searchable: true
 
 A **User** is an account that any of your users can have. Each user is assumed to have a unique email address. You can define properties for all users in the user.yml file.
 
-**User Profiles** are roles in the marketplace. Each User Profile can be associated with any number of Properties, Categories, and Custom Model Types.
-
-## The default User Profile
-
-All users are assigned a User Profile named Default. If you don't want to distinguish one type of user from the other, you don't need to create any other user profiles.
+**User Profiles** are roles in the marketplace. Each User Profile can be associated with any number of Properties, Images, Attachments and Custom Model Types.
 
 ## Sharing attributes across multiple profiles
 
-If your multiple profiles happen to have the same Properties, you can extract them to a container profile which will group them and could be used in different profiles.
+If your multiple profiles happen to have the same Properties, you can extract them to a container profile which will group them.
 
 ### Example
 
-A company has a lot of employees, but all of them have the same attributes like name, email, emergency contact, so it would be a good idea to make a default profile for employee, and then use more specific ones for specific roles in the company.
+A company has a lot of employees, but all of them have the same attributes like name, email, emergency contact, so it would be a good idea to make a profile for employee, and then use more specific ones for specific roles in the company.
+
+```yaml
+# user_profil_types/employee.yml
```

Comment:
Convention:

> #### file/path/name.ext
> ```liquid
> ...
> ```

Reply:
Thank you! Updated based on your feedback.